ccm_web/client/src/pages/FormatThirdPartyGradebook.tsx

```diff
@@ -364,10 +364,12 @@ export default function FormatThirdPartyGradebook (props: FormatThirdPartyGradeb
     processedRecords: GradebookUploadRecord[], assignmentHeader: string, file: File
   ): JSX.Element => {
     const recordsToReview = processedRecords.map((r, i) => ({ rowNumber: i + 2, ...r }))
-    const dataToDownload = 'data:text/csv;charset=utf-8,' + csvParser.createCSV<GradebookUploadRecord>({
-      fields: [...REQUIRED_ORDERED_HEADERS, assignmentHeader],
-      data: processedRecords
-    })
+    const dataToDownload = 'data:text/csv;charset=utf-8,' + encodeURIComponent(
```

Comment:
Optional: Besides the change, I feel the text with charset we can take it as constant. So that we just pull it.